GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -210,18 +210,14 @@ private string DescribeRevision([CanBeNull] ObjectId objectId, int maxLength = 0
         /// Provide a description for the first selected or parent to the "primary" selected last
         /// </summary>
         /// <returns>A description of the selected parent</returns>
-        private string DescribeSelectedParentRevision()
+        private string DescribeRevision(IEnumerable<GitRevision> parents)
```

Comment:
```suggestion
        private string DescribeRevision(List<GitRevision> parents)
```

and replace `parents.Count()` with `parents.Count`